client/src/cli.rs

```diff
@@ -183,54 +183,30 @@ pub struct BandadaData {
 	address: Option<String>,
 }
 
-/// Bandada API action.
-pub enum Action {
-	Add,
-	Remove,
-}
-
-impl FromStr for Action {
-	type Err = &'static str;
-
-	fn from_str(s: &str) -> Result<Self, Self::Err> {
-		match s {
-			"add" => Ok(Action::Add),
-			"remove" => Ok(Action::Remove),
-			_ => Err("Invalid action."),
-		}
-	}
-}
```

Comment:
I like this idea, why remove it?

Reply:
Yes I liked it too just wasn't 100% sure about it, but if you like it then I'll add it again 😆